addons/web/static/src/core/domain_selector/domain_selector_fields.js

```diff
@@ -133,10 +132,11 @@ function parseValue(fieldType, value) {
     }
 }
 
-function makeEditor(component, props) {
+function makeEditor(component, props, isSupported) {
```

Comment:
Looks like that `props`, `isSupported`, `shouldResetValue` are optional info
```js
function makeEditor(component, options = {}) {
  return {...};
}
```

Reply:
done